packages/theme-chalk/src/table.scss

```diff
@@ -32,6 +32,14 @@
     }
   }
 
+  &.#{$namespace}-table--footer {
+    @include e(inner-wrapper) {
+      &::before {
+        bottom: 1px;
+      }
+    }
+  }
+
```

Comment:
**Question:** table footer seems a element for me, so should we use `el-table__footer` instead?

Reply:
> **Question:** table footer seems a element for me, so should we use `el-table__footer` instead?

I've changed to `has-footer`,